### ISSUE TITLE
Error: googleapi: Error 400: No change in configuration. Must specify a change to configuration.recovery_configuration.scheduled_snapshots_configuration, badRequest

### ISSUE THREAD

**Issue by tooj-cuure** [terraform-google-modules/terraform-google-composer#72]
### TL;DR

I try to manage(import) existing Cloud Composer environment using `terraform-google-modules/composer/google//modules/create_environment_v2`. The environment import was quite done unless when I do a `terraform plan` I see to modification on the resource

```
 ~ config {
            # (5 unchanged attributes hidden)

          - recovery_config {
              - scheduled_snapshots_config {
                  - enabled = false -> null
                }
            }

          ~ workloads_config {

              - triggerer {
                  - count     = 0 -> null
                  - cpu       = 0 -> null
                  - memory_gb = 0 -> null
                }

                # (3 unchanged blocks hidden)
            }

            # (6 unchanged blocks hidden)
        }

```
Those 2 features are disabled on the existing Cloud composer environement. There is no variable to configure these 2 blocks on the terraform module. Otherwise the `triggerer` block didn't cause issue when executing `terraform apply`. But the `configuration.recovery_configuration.scheduled_snapshots_configuration` cause this issue
```
╷
│ Error: googleapi: Error 400: No change in configuration. Must specify a change to configuration.recovery_configuration.scheduled_snapshots_configuration, badRequest
│ 
│   with module.composer.google_composer_environment.composer_env,
│   on .terraform/modules/composer/modules/create_environment_v2/main.tf line 27, in resource "google_composer_environment" "composer_env":
│   27: resource "google_composer_environment" "composer_env" {
│ 
╵

```

**Proposed solution:** Maybe you should set a default values to those blocks or allow to configure them.

### Terraform Configuration

```hcl
module "composer" {
  source            = "terraform-google-modules/composer/google//modules/create_environment_v2"
  version           = "3.4.0"
  project_id        = var.project_id
  composer_env_name = var.composer_env

  image_version = local.composer_image_version
  network       = google_compute_network.vpc.name
  subnetwork    = google_compute_subnetwork.subnet.name

  environment_size = var.composer_env_size

  maintenance_recurrence = local.maintenance_recurrence

  pypi_packages = local.pypi_packages

  web_server = {
    "cpu" = var.workload_cpu
    "memory_gb" = var.workload_memory_gb
    "storage_gb" = var.workload_storage_gb
  }

  scheduler = {
    count      = var.scheduler_count
    cpu        = var.workload_cpu
    memory_gb  = var.workload_memory_gb
    storage_gb = var.workload_storage_gb
  }

  worker = {
    "cpu" = var.workload_cpu
    "memory_gb" = var.workload_memory_gb
    "max_count" = var.worker_max_count
    "min_count" = var.worker_min_count
    "storage_gb" = var.workload_storage_gb
  }
}
```

### Terraform Version

```sh
Terraform v1.3.6
```
